### PR TITLE
T/retries+readme

### PR DESCRIPTION
**Pull request by bluemaex**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,37 @@
...
+- Include as a dependency in your app: `angular.module('myApp', ['http-watch'])`
+
+Manual
+------
+
+This module installs $http interceptor and provides the `httpWatcher` module.
+
+The $http interceptor does the following:
+
+If an HTTP request fails, the event `network:http` is broadcasted with the configuration object (this is the requested URL, payload and parameters)
```
Comment (sashless) [inline README.md]: `network:http-error`

File: README.md [hunk 2]
```diff
@@ -1 +1,37 @@
...
+of said request. If the HTTP Error-Code is either `0` or `401` it is buffered and can be replayed at any given time using the `continue()` method of the `httpWatcher` module.
+You are responsible to invoke this method after you handled the error.
```
Comment (sashless) [inline README.md]: how to call `continue` ?

Comment (bluemaex) [inline README.md]: As stated above:
> This module installs $http interceptor and provides the `httpWatcher` module.

example:
```
return function(httpWatcher, $translate) {
    httpWatcher.continue();
});
```

File: README.md (hunk 1 above)
Comment (bluemaex) [inline README.md]: Hehe, good point.

File: README.md [hunk 3]
```diff
@@ -1 +1,45 @@
...
+------
+
+- Install via npm: `npm install --save imagineeasy-angular-http-watcher`
+- Include as a dependency in your app: `angular.module('myApp', ['http-watch'])`
+
+Manual
+------
+
+This module installs $http interceptor and provides the `httpWatcher` module.
+
```
Comment (sashless) [inline README.md]: its not a module, its a service. The module is `http-watch`

Comment (sashless) [inline README.md]: or a factory, provider, value etc but its not a module speaking in angular terms

**Pull request merged**